src/main/java/com/gistpetition/api/petition/application/PetitionService.java

```diff
@@ -64,6 +66,26 @@ public Long createPetition(PetitionRequest petitionRequest, Long userId) {
         return PetitionPreviewResponse.pageOf(petitionRepository.findAllByCategoryAndReleasedTrue(Category.of(categoryId), pageable));
     }
 
+    @Transactional(readOnly = true)
+    public Page<PetitionPreviewResponse> retrieveReleasedAndExpiredPetition(Pageable pageable) {
```

Comment:
```suggestion
    public Page<PetitionPreviewResponse> retrieveExpiredPetition(Pageable pageable) {
```
여기도 마찬가지로 만료된 청원 정도의 메시지면 좋을 것 같아요